packages/process/_test.pony

```diff
@@ -95,6 +120,39 @@ class iso _TestFileExec is UnitTest
   fun timed_out(h: TestHelper) =>
     h.complete(false)
 
+  fun _setup_notifier(h: TestHelper, path: FilePath): ProcessNotify iso^ =>
+    recover object is ProcessNotify
```

Comment:
I wanted to stick to the style that @Theodus had already established. We could do another pull to change it but that felt extraneous to this.